addon/ajax.ts

```diff
@@ -0,0 +1,14 @@
+
+import fetch from 'fetch';
+
+export default async function ajax(
+  input: RequestInfo,
+  init?: RequestInit
+): Promise<Response> {
+  return fetch(input, init).then(response => {
```

Comment:
I would either

A) leave `ajax` as an `async` function, and use `await` to wait for promises instead of `.then`
B) treat `ajax` as a regular (non-`async`) function and use `.then` to wait for promises instead of `await`

Currently you're mixing (A) and (B) by making the function `async` but never `await`ing

Reply:
oooh I was in the middle of transforming it to async but decide to leave it as is, because we are going to have a new [service](https://github.com/emberjs/ember.js/issues/17476) and probably deprecate this.